### PR TITLE
Fix url encoding in DefaultHttpJsonRequest

### PR DESCRIPTION
**Pull request by akorneta**
@skabashnyuk @evoevodin 
UriBuilder.build()  performs encode it was necessary to get rid of second encoding of query params

### CODE REVIEW

File: core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/DefaultHttpJsonRequest.java [hunk 1]
```diff
@@ -34,7 +34,6 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriBuilder;
-
```
Comment (voievodin) [inline core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/DefaultHttpJsonRequest.java:37]: Why did you remove this line ? [spec](https://github.com/codenvy/codenvy-dev-resources/wiki/Source-file#imports)

File: core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/DefaultHttpJsonRequest.java [hunk 2]
```diff
@@ -201,11 +199,7 @@ DefaultHttpJsonResponse doRequest(int timeout,
 
             if (hasQueryParams) {
                 for (Pair<String, ?> parameter : parameters) {
-                    String name = URLEncoder.encode(parameter.first, "UTF-8");
-                    String value = parameter.second == null ?
-                                   null :
-                                   URLEncoder.encode(String.valueOf(parameter.second), "UTF-8");
-                    ub.queryParam(name, value);
+                    ub.queryParam(parameter.first, parameter.second == null ? null : String.valueOf(parameter.second));
```
Comment (voievodin) [inline core/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/DefaultHttpJsonRequest.java]: If you put `parameter.second` to the `ub.queryParam` the behaviour will be the same as with ternary operator

**Comment by skabashnyuk** [comment]
ок

**Comment by garagatyi** [comment]
@akorneta Please run selenium test to ensure everything will work as expected.

**Comment by voievodin** [comment]
ok
